web/src/stream_create_subscribers.ts

```diff
@@ -4,46 +4,48 @@ import render_new_stream_user from "../templates/stream_settings/new_stream_user
 import render_new_stream_users from "../templates/stream_settings/new_stream_users.hbs";
 
 import * as add_subscribers_pill from "./add_subscribers_pill";
+import type {InputPillContainer} from "./input_pill";
 import * as ListWidget from "./list_widget";
 import * as people from "./people";
 import {current_user} from "./state_data";
 import * as stream_create_subscribers_data from "./stream_create_subscribers_data";
+import type {CombinedPillItem} from "./typeahead_helper";
 import * as user_sort from "./user_sort";
 
-let pill_widget;
-let all_users_list_widget;
+let pill_widget: InputPillContainer<CombinedPillItem>;
+let all_users_list_widget: ListWidget.ListWidget<number, people.User>;
```

Comment:
`ListWidget.ListWidget` looks a bit weird in terms of readability, can we import the type separately?

Reply:
What about `ListWidgetType`? This naming convention is used in `dropdown_widget.ts`

Reply:
Cool!! Thanks